### PR TITLE
FIX Authentication error identified as connection one

### PR DESCRIPTION
**Pull request by dmoranj**
Fix the type of error to raise in case of a failed user authentication (from connection error to authentication error).

### CODE REVIEW

**Comment by dmoranj** [comment]
Fix #174

File: lib/services/keystoneAuth.js [hunk 1]
```diff
@@ -207,7 +207,7 @@ function retrieveUser(req, callback) {
                 innerCb(new errors.KeystoneAuthenticationError(message));
```
Comment (fgalan) [inline lib/services/keystoneAuth.js]: CHANGES_NEXT_RELEASE entry missing about the closing of #174 ?

Comment (dmoranj) [inline lib/services/keystoneAuth.js]: Fixed in 7156de6

**Comment by fgalan** [comment]
LGTM (passing the ball to @jcalderin for final LGTM + merge + delete)

**Comment by jcalderin** [comment]
LGTM

**Pull request merged**